airflow/contrib/operators/app_engine_operator.py

```diff
@@ -75,6 +75,8 @@ def execute(self, context):
         # It seems that when an operator returns, it is considered successful,
         # and an operator fails if and only if it raises an AirflowException.
         # Good luck finding documentation saying that though.
+        if self.command_params.get('skip') is True:
+            return
```

Comment:
This assumes that there will never be a command with `skip` as a parameter. I don't think this is what we want to do here.